index.js

```diff
@@ -162,7 +162,9 @@ Ycb.prototype = {
             }
         }
 
-        this._applySubstitutions(config);
+        if (!options.noSubstitutions) {
```

Comment:
its always weird to have a negative key but positive value to enable it. Can we make it `applySubstitutions` and make the default `true` to maintain BC?